### PR TITLE
GUVNOR-2637: Remove Inbox feature

### PR DESCRIPTION
**Pull request by karreiro**
See https://issues.jboss.org/browse/GUVNOR-2637

### CODE REVIEW

**Comment by karreiro** [comment]
Jenkins, please retest this.

File: kie-wb/kie-wb-webapp/src/main/resources/org/kie/workbench/client/resources/i18n/AppConstants.properties [hunk 1]
```diff
@@ -31,9 +31,6 @@ repositories=Repositories
 listRepositories=List
 cloneRepository=Clone repository
 newRepository=New repository
-inboxIncomingChanges=Incoming changes
-inboxRecentlyEdited=Recently edited
-inboxRecentlyOpened=Recently opened
```
Comment (Rikkola) [inline kie-wb/kie-wb-webapp/src/main/resources/org/kie/workbench/client/resources/i18n/AppConstants.properties]: Nice, you even picked up these.

**Comment by Rikkola** [comment]
+1

**Comment by karreiro** [comment]
Jenkins, please retest this.

**Comment by csadilek** [comment]
@karreiro @Rikkola The inbox dependencies should also be removed from the guvnor-bom: https://github.com/droolsjbpm/droolsjbpm-build-bootstrap/blob/master/guvnor-bom/pom.xml#L292

**Comment by karreiro** [comment]
Thanks @csadilek https://github.com/droolsjbpm/droolsjbpm-build-bootstrap/pull/253

**Pull request merged**